pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -220,11 +223,185 @@ def _get_fourier_models_data(self) -> pd.DataFrame:
             n_order=self.yearly_seasonality,
         )
 
+    def channel_contributions_forward_pass(
+        self, channel_data: Union[pd.DataFrame, pd.Series, npt.NDArray[np.float_]]
+    ) -> npt.NDArray[np.float_]:
+        """Evaluate the channel contribution for a given channel data and a fitted model, ie. the forward pass.
+
+        Parameters
+        ----------
+        channel_data : array-like
+            Input channel data.
+
+        Returns
+        -------
+        array-like
+            Transformed channel data.
+        """
+        alpha_posterior = (
+            az.extract(self.fit_result, group="posterior", var_names=["alpha"])
+            .to_numpy()
+            .T
+        )
+
+        lam_posterior = (
+            az.extract(self.fit_result, group="posterior", var_names=["lam"])
+            .to_numpy()
+            .T
+        )
+        lam_posterior_expanded = np.expand_dims(a=lam_posterior, axis=1)
+
+        beta_channel_posterior = (
+            az.extract(self.fit_result, group="posterior", var_names=["beta_channel"])
+            .to_numpy()
+            .T
+        )
+        beta_channel_posterior_expanded = np.expand_dims(
+            a=beta_channel_posterior, axis=1
+        )
```

Comment:
Nitpick, but maybe more readable?

```suggestion
        beta_channel_posterior_expanded = np.expand_dims(
            a=beta_channel_posterior, axis=-1
        )
```

Reply:
I think it should be ok. I expect to have at least two channels in 99% of mmm applications (otherwise what do we need to optimize).